### DISCUSSION TITLE
Welcome to Iris Discussions!

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4068] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bjlittle** [comment]
If you're new to GitHub Discussions... and well, I certainly am :smile:

Then why not head over to the official [GitHub Discussions](https://docs.github.com/en/discussions) documentation to get the latest scoop on what this new cool GitHub beta feature is all about.

It's a worthwhile read, and it might help you get the most out of our shiny new Iris community Discussions :+1:

**Reply by trexfeathers** [reply]
Can we ditch Google Groups?! One less place?! One less tool?!

**Reply by tkknight** [reply]
Certainly an option.  Raised in other issues too, see https://github.com/SciTools/iris/issues/3905, https://github.com/SciTools/iris/issues/3860.  

May need a little more thought, review existing googlegroups use, consider how to close it down and redirect here, what guidelines we want in place for this discussion area etc.
